src/server/ghService/helpers/packageImageHelper.ts

```diff
@@ -0,0 +1,36 @@
+import { isNullOrUndefined } from "util";
+
+import fetch from "node-fetch";
+import { URL } from "url";
+
+const getFavicon = async (url?: string): Promise<string> => {
+  if (isNullOrUndefined(url) || url === "") {
```

Comment:
can use `x == null` to check for null/undefined

Reply:
done